### DISCUSSION TITLE
Is this still under development?

### DISCUSSION THREAD

**Discussion by ObiWahn** [debauchee/barrier#2012] [General]
I used synergy years back and just found barrier as on open source fork. Now I wonder if it is worth looking at, because it seems that has not been development for years. On the other hand mouse / keyboard drivers probably do not change so frequently. If there is some happy user or a maintainer I would be happy to hear about the state.

https://github.com/input-leap/input-leap <-- probably this is what we are looking for:P

**Comment by moritztim** [comment]
I've used it on a multitude of different devices and almost every time when there was some issue it was my fault. Usually works like a charm.

**Reply by moritztim** [reply]
The input leap team also recommends using barrier for now.
> Our advice is to stick with Barrier v2.4.0/v2.3.4 for the time-being.

https://github.com/input-leap/input-leap?tab=readme-ov-file#faq---frequently-asked-questions

**Comment by moritztim** [comment]
https://github.com/input-leap/input-leap/issues/1414
